### ISSUE TITLE
"View Other" is flaky in a few situations

### ISSUE THREAD

**Issue by timleavitt** [intersystems-community/vscode-objectscript#348]
I've had trouble with "View Other" in the following situations where it should reasonably work:
* Workspace configuration issues (e.g., using isfs with server manager rather than objectscript.conn, and objectscript.conn not fully specified - e.g., missing ns)
* Connecting to multiple remote servers and trying to "view other" on a non-default remote server
* Viewing other for CSP files under isfs

Bottom line is that the "view other" command uses the current workspace context, rather than the file in question (with a URI that may point to a non-default server via isfs). My solution to #347 includes a reusable piece that helps address this as well.

(I'm planning to address this via PR this week.)
